docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+services:
+  redis:
+    image: "redis:alpine"
+    restart: always
+    expose:
+      - 6379
+    ports:
+      - "6379:6379"
+  db:
+    image: postgres:latest
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
```

Comment:
if this the local docker compose then I'm totally fine with this but we should also add a docker-compose.ci.yml for the secrets